src/main/resources/templates/editEvent.html

```diff
@@ -91,26 +91,7 @@ <h2>Update Event</h2>
     <hr>
 
     <a th:href="@{/}"> Back to the Events List</a>
-    <script>
-        const timeControl = document.querySelector('input[type="datetime-local"]');
-        const timeControlHidden = document.querySelector('.hid');
-
-        window.onload = (event) => {
-            console.log('page is fully loaded');
-            console.log(timeControlHidden.value);
-            console.log(timeControlHidden.value.replace(' ', 'T'));
-            timeControl.value = timeControlHidden.value.replace(' ', 'T');
-        };
-
-        timeControl.addEventListener('change', (event) => {
-            console.log(timeControl);
-            console.log(timeControl.value);
-            let date = new Date(timeControl.value);
-            console.log(date.toISOString().slice(0, 19).replace('T', ' '));
-            timeControlHidden.value = date.toISOString().slice(0, 19).replace('T', ' ');
-            console.log(timeControlHidden.value);
-        });
-    </script>
+    <script src="/js/event.js"></script>
```

Comment:
Generally, it is a good idea to have your JS and HTML seperate, so I moved this Javascript code to its own file.

Reply:
Yes, good idea.